_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
![IMG-20211121-WA0008](https://user-images.githubusercontent.com/95079525/150138005-7b436688-eba0-4745-bf67-1e12274aef1d.jpg)


Its me Sidharth Prem, an software development aspirant and  programming enthusiast, studying as a first year B.Tech CSE student at Marian Engineering student.